pytorch_lightning/trainer/trainer.py

```diff
@@ -1091,17 +1091,16 @@ def _run(
         if hasattr(model, "hparams"):
             parsing.clean_namespace(model.hparams)
 
-        verify_loop_configurations(self, model)
-
-        # attach model log function to callback
-        self._callback_connector.attach_model_logging_functions(model)
-
         # attach model to the training type plugin
         self.training_type_plugin.connect(model)
 
+        self._callback_connector._attach_model_callbacks()
+        self._callback_connector._attach_model_logging_functions()
+
+        verify_loop_configurations(self, model)
```

Comment:
thanks, @carmocca,
I guess with the changes you made we can remove the model from `verify_loop_configurations` as well?